src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProviderPhone.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutDirSuffix>Phone\</OutDirSuffix>
+    <ProjectGuid>{6A4C8A1D-4C47-4191-9681-136FC3B7209C}</ProjectGuid>
```

Comment:
I'd be curious to know what shows up in the .sln file (whether VS cracks the project and reads this directly or uses the evaluated property from MSBuild).  If it's the former, it might not matter that this is overwritten.  If the latter, then yeah, we need a new property group below the Import...